runtime/pkg/email/email.go

```diff
@@ -60,9 +61,14 @@ func (c *Client) SendScheduledReport(opts *ScheduledReport) error {
 	// Build subject
 	subject := fmt.Sprintf("%s (%s)", opts.Title, data.ReportTimeString)
 
+	var err error
 	// Resolve template
 	buf := new(bytes.Buffer)
-	err := c.templates.Lookup("scheduled_report.html").Execute(buf, data)
+	if opts.External {
+		err = c.templates.Lookup("scheduled_report_external.html").Execute(buf, data)
+	} else {
+		err = c.templates.Lookup("scheduled_report.html").Execute(buf, data)
+	}
```

Comment:
Did you experiment with using something like `{{ if .OpenLink }}` in the `scheduled_report.mjml` instead of having a separate file? It's slightly tricky because we pre-compile the MJML, so the templating can't impact the HTML structure, but maybe something like this would work?
```html
<mjml>
  <mj-head>
    <mj-style> 
      .hidden {
          display: none !important;
          mso-hide: none;
      }   
    </mj-style>
  </mj-head>
  <mj-body>
    <mj-section>
      <mj-column>
        <mj-button css-class="{{ if empty .OpenLink }} hidden {{ end }}">
          Don't click me!
        </mj-button>
      </mj-column>
    </mj-section>
  </mj-body>
</mjml>
```

Reply:
awesome it worked! made the change. also removed sending open url from get report metadata call.